util/encoding/encoding_test.go

```diff
@@ -367,6 +367,30 @@ func TestDecodeInvalid(t *testing.T) {
 			pattern: "unknown escape",
 			decode:  func(b []byte) error { _, _, err := DecodeBytesDescending(b, nil); return err },
 		},
+		{
+			name:    "Float, no terminator",
+			buf:     []byte{floatPosLarge},
+			pattern: "did not find terminator",
+			decode:  func(b []byte) error { _, _, err := DecodeFloatAscending(b, nil); return err },
+		},
+		{
+			name:    "FloatDecreasing, no terminator",
```

Comment:
While making this change I realized that we use the term "Decreasing" throughout this `encoding_test.go` file to mean what we now call "Descending". I kept the convention here, but wanted to bring it up for a potential different PR. Should we make the change globally in this file from "Decreasing" to "Descending", keeping in mind that this will change benchmark names and thus mess up historic benchmark comparisons?